core/controller/src/main/scala/whisk/core/controller/actions/SequenceActions.scala

```diff
@@ -353,6 +369,24 @@ protected[actions] trait SequenceActions {
 
     /** Max atomic action count allowed for sequences */
     private lazy val actionSequenceLimit = whiskConfig.actionSequenceLimit.toInt
+
+    /** Starts the tracing for the sequence action and creates the tracing metadata hash map into which
+     *  the tracer implementation  inserts its own ids, these ids are then propagated via the ActivationMessage into
+     *  invoker that can add the reference from the primitive action to the parent sequence action
+     */
+    private def startTracingHelper(cause: Option[ActivationId],
```

Comment:
I wasn't aware of the markers in the akka logging system. Do I get it right that you [use it](https://github.com/apache/incubator-openwhisk/blob/master/common/scala/src/main/scala/whisk/common/Logging.scala#L125) for calculating how long some execution took?